Acceptable Usage Policy.md

```diff
@@ -18,28 +18,30 @@ Online communities have the potential to change people's lives, so we want to en
-- Sharing any content promoting, encouraging or glorifying self-harm or suicide. This includes any content that pushes people to cut or injure themselves; that embraces eating disorders such as anorexia or bulimia; or that encourages people to commit suicide as opposed to referring them to the correct treatment and help. If you notice someone in need of urgent help, **please contact your local authorities**.
-- Attempts to hack or otherwise gain unauthorised access to our service - for example, accessing another person's account without permission, or attempting to steal another person's account. This also includes attempts to gain access to our backend networks. **Please contact us ahead of time for permission to test our networks by emailing [security@revolt.chat](mailto:security@revolt.chat).**
-- Distributing malware such as viruses, trojans, worms, etc. Any software designed with malicious intent to attack another person or to steal their data is not tolerated.
-- Sharing content that is designed to harass or degrade someone else, or that can directly threaten someone's physical, mental or financial state. This includes death threats, doxxing (sharing personal information without permission), or ignoring someone's privacy.
+- **Sharing any content or imagery that is illegal**. Includes but is not limited to, sharing any pornography that is subject to legal age restrictions, pornography that depicts minors, revenge pornography, the selling or facilitating the usage of drugs, extortion or blackmail.
+- **Sharing any content promoting, encouraging or glorifying self-harm or suicide**. This includes any content that pushes people to cut or injure themselves; that embraces eating disorders such as anorexia or bulimia; or that encourages people to commit suicide as opposed to referring them to the correct treatment and help. If you notice someone in need of urgent help, _please contact your local authorities_.
+- **Attempts to hack or otherwise gain unauthorised access to our service**. For example, accessing another person's account without permission, or attempting to steal another person's account. This also includes attempts to gain access to our backend networks. _Please contact us ahead of time for permission to test our networks by emailing [security@revolt.chat](mailto:security@revolt.chat)._
+- **Distributing malware such as viruses, trojans, worms, etc**. Any software designed with malicious intent to attack another person or to steal their data is not tolerated.
+- **Sharing content that is designed to harass or degrade someone else, or that can directly threaten someone's physical, mental or financial state**. This includes death threats, doxxing (sharing personal information without permission), or ignoring someone's privacy.
```

Comment:
Should it be spelt doxxing or doxing? I've more commonly seen it spelt doxing.

Reply:
I'd say I've seen more of the former? It feels more correct to me for some reason but hmm

Reply:
Doxing is defined in Oxford English Dictionary so I'll swap it to that